packages/devtools_app/lib/src/app.dart

```diff
@@ -516,6 +518,55 @@ class SettingsDialog extends StatelessWidget {
             toggle: preferences.toggleVmDeveloperMode,
             gaItem: gac.vmDeveloperMode,
           ),
+          Column(
```

Comment:
put this in a helper widget _VerboseLoggingSetting. This build method is starting to get large